doc/user/ospfd.rst

```diff
@@ -310,6 +310,16 @@ To start OSPF process you have to specify the OSPF router.
    In some cases it may be more convenient to enable OSPF on a per
    interface/subnet basis (:clicmd:`ip ospf area AREA [ADDR]`).
 
+.. index:: proactive-arp
+.. clicmd:: proactive-arp
+
+.. index:: no proactive-arp
+.. clicmd:: no proactive-arp
+
+   This command enables or disables sending ARP requests to update neighbor
+   table entries. It speeds up convergence for /32 networks on a p2p
```

Comment:
Nit: maybe we should use upper case for `p2p`? Assuming it means Peer to Peer.

Also I think it would be nice to document that `proactive-arp` is the default.

Reply:
Thanks for the suggestions! 
I updated the docs.